File: abc/abc168/a.cpp
// https://atcoder.jp/contests/abc168/submissions/26109145

#include <bits/stdc++.h>
using namespace std;
int main() {
  int N;
  cin >> N;
  int X = N % 10;
  if(X == 0 || X == 1 || X == 6 || X == 8) cout << "pon";
  else if(X == 3) cout << "bon";
  else cout << "hon";
}

File: abc/abc168/a.java
// https://atcoder.jp/contests/abc168/submissions/41616723

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    n %= 10;

    if(n == 2 || n == 4 || n == 5 || n == 7 || n == 9) {
      System.out.println("hon");
    } else if(n == 0 || n == 1 || n == 6 || n == 8) {
      System.out.println("pon");
    } else {
      System.out.println("bon");
    }
  }
}
